### ISSUE TITLE
[Feature Request] Remove all user dictionary

### ISSUE THREAD

**Issue by azzamsa** [AnySoftKeyboard/AnySoftKeyboard#1379]
Can we mark all user-dictionary words and delete it completely ?

I tried to remove UserWords.xml but the saved words still there.

Thank you for AnySoftKeyboard :)

**Comment by menny** [comment]
The best that I can offer right now is to backup everything except the User-Words and then clear-data of the app

**Comment by azzamsa** [comment]
would you like to point us. What directory to delete in order to remove all ASK data ?

**Comment by azzamsa** [comment]
I have tried to uninstall it several times with different methods. No one works. the user dictionary still persist.

I don't find any left data in `Android/data/` but I can't figure it out why I can completely remove user dictionary.

**Comment by azzamsa** [comment]
I have tried to trick the app. using `restore` with deleted wordlist

```
<?xml version="1.0" encoding="utf-8"?>
<userwordlist>
    <wordlist locale="en">
        <w f="255">text</w>
        <w f="255">dan</w>
.....
```

But it seems the app use new word list format. So I changed that. But it still doesn't work. `restoring` words doesn't delete the old one

**Comment by azzamsa** [comment]
`delete data` and `delete cache` in Android setting then reinstalling also doesn't work. Maybe I need a refresher.

**Comment by azzamsa** [comment]
I reset my android phone because of other problem. and all this user dictionary gone.

**Comment by karthikiyengar** [comment]
Yeah I can confirm that the clear data doesn't get rid of the dictionary somehow.

Tried deleting the data folder, no luck. How do we flush the user dictionary without having to reset the device?

**Comment by nils-se** [comment]
I stumbled upon this issue and solved it with a workaround: It seems the user dictionary is not managed by AnySoftKeyboard itself, but centralized by Android. So I used the app "User Dictionary Manager (UDM)" and was able to delete all words in my user dictionary in AnySoftKeyboard after I reset the app. I hope this info may help anybody searching for a quick fix. I love AnySoftKeyboard, thanks for the great work everyone :)
